openghg/standardise/meta/_attributes.py

```diff
@@ -156,15 +161,15 @@ def get_attributes(
 
     # Load attributes files
     species_attrs = get_species_info()
-    attributes_data = load_json("attributes.json", internal_data=True)
+    attributes_data = load_json(filename="attributes.json")
```

Comment:
Moving over discussion from PR #554 as these questions are more relevant here since this where the original changes seem to have been made:

    rt17603 (3 weeks ago)
    Does this still run with the internal_data flag removed? This was to flag the data was within our openghg/data
    folder and so was needed for e.g. "icos_atmos_site_metadata.json" as this isn't a full path to the data file?

    gareth-j (3 weeks ago)
    Yeah I think we both made changes to that function at a similar time. I added the path kwarg so you could load
    any file, and you added the internal_data flag. Maybe we could more easily differentiate between them by 
    creating a load_internal_json function and another load_json that just takes a full filepath? Quite a few bugs 
    were introduced when I merged in both our changes.

    rt17603 
    Yes we could approach this by using separate functions - that way it's still clear where the data is coming from
    (and what is internal to openghg).

Reply:
No, shall I make those changes before this goes in?

Reply:
Yes I think we should make sure those changes go in here.

Reply:
OK cool, I'll do that now.